app/services/video-encoding-optimizations/video-encoding-optimizations.ts

```diff
@@ -163,9 +165,14 @@ export class VideoEncodingOptimizationService extends PersistentStatefulService<
       bitrate: currentSettings.streaming.bitrate,
     };
 
+    // change the resolution only if user didn't set a custom one
     if (!currentSettings.streaming.hasCustomResolution) {
-      // change the resolution only if user didn't set a custom one
-      newStreamingSettings.outputResolution = encoderProfile.resolutionOut;
+      // skip this output res step for tiktok since they stream in portrait
+      if (this.userService.platformType === 'tiktok') {
```

Comment:
i feel like you can just add this condition to the first if statement

Reply:
u rite, pushed a change